src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -507,13 +507,36 @@ module Types = struct
       type t =
         { total: Balance.t
         ; unknown: Balance.t
+        ; timing: Mina_base.Account_timing.t
         ; breadcrumb: Transition_frontier.Breadcrumb.t option }
 
+      let min_balance (b : t) =
+        match (b.timing, b.breadcrumb) with
+        | Untimed, _ ->
+            Balance.zero
+        | Timed _, None ->
+            Balance.zero (* TODO: What should we do when no breadcrumb? *)
```

Comment:
Can you make this “nullable” (an option)? And make null the state with no breadcrumbs to distinguish between fully unlocked. And then please specify that in the docs as well for those liquid/locked fields